ios/ExpensifyCash/Chat.entitlements

```diff
@@ -6,9 +6,8 @@
 	<string>development</string>
 	<key>com.apple.developer.associated-domains</key>
 	<array>
-		<string>applinks:www.expensify.cash</string>
 		<string>applinks:staging.expensify.cash</string>
```

Comment:
```suggestion
		<string>applinks:staging.new.expensify.com</string>
```

Reply:
Can you make this change on android too? Thanks: https://github.com/Expensify/App/blob/main/android/app/src/main/AndroidManifest.xml#L58

Reply:
Hello, the requested changes have been implemented.